### PR TITLE
add domains and fix help messages

### PR DESCRIPTION
**Pull request by alexz429**
# Related Issues
Resolves https://github.com/uwcsc/codeybot/issues/38

# Summary of Changes 
Added new domains, and fixed domain help messages to be code blocks

# Steps to Reproduce
image

### CODE REVIEW

File: src/components/interview.ts [hunk 1]
```diff
@@ -29,9 +33,9 @@ export const getInterviewer = async (id: string): Promise<Interviewer | undefine
   return await db.get('SELECT * FROM interviewers WHERE user_id = ?', id);
 };
 
-export const getDomainsString = (domains: string[]): string => _.join(domains, ', ');
+export const getDomainsString = (domains: string[]): string => '`' + _.join(domains, '`, `') + '`';
```
Comment (gzcharleszhang) [inline src/components/interview.ts:36]: nit: use template string

Comment (alexz429) [reply src/components/interview.ts:36]: is it actually better in this case? I have to escape the backticks which make it look really awkward.

Comment (gzcharleszhang) [reply src/components/interview.ts]: I think in general we want to use template strings as much as possible. In this case the backticks made it uglier than normal but we should try to be consistent.

**Review by gzcharleszhang — APPROVED** [review]

File: src/components/interview.ts [hunk 2]
```diff
@@ -9,8 +9,12 @@ export const availableDomains: { [key: string]: string } = {
   backend: 'Backend',
   design: 'Design',
   pm: 'PM',
-  datasci: 'Datasci',
-  infra: 'Infra'
+  datasci: 'DataSci',
+  infra: 'Infra',
```
Comment (kallentu) [inline src/components/interview.ts:13]: Stupid nit: Can we make this alphabetical? So it's easier to add/remove from and probably easier for the user to find their domain.

**Pull request merged**